pkgs/applications/science/biology/spades/default.nix

```
@@ -17,6 +17,11 @@ stdenv.mkDerivation rec {
 
   sourceRoot = "${pname}-${version}/src";
 
+  CXXFLAGS = [
```

```suggestion
  env.CXXFLAGS = toString [
```

I'd prefer that, but certainly not a blocker for merging.

---

Sorry. Will update.